doc/source/file-io.rst

```diff
@@ -0,0 +1,78 @@
+File formats and input/output
+*****************************
+
+In addition to the data management features provided by :doc:`api-backend`, ixmp is able to write and read :class:`TimeSeries` and :class:`Scenario` data to and from files.
+This page describes those options and formats.
+
+Time series data
+================
+
+Time series data can be:
+
+- Read using :meth:`.import_timeseries`, or the CLI command ``ixmp import timeseries FILE`` for a single TimeSeries object.
+- Written using :meth:`.export_timeseries_data` for multiple TimeSeries objects at once.
+
+Both CSV and Excel files in the IAMC time-series format are supported.
+
+.. _excel-data-format:
+
+Scenario/model data
+===================
+
+Scenario data can be read from/written to Microsoft Excel files using :meth:`.Scenario.read_excel` and :meth:`.to_excel`, and the CLI commands ``ixmp import scenario FILE`` and ``ixmp export FILE``.
+The files have the following structure:
+
+- One sheet named 'ix_type_mapping' with two columns:
+
+  - 'item': the name of an ixmp item.
+  - 'ix_type': the item's type as a length-3 string: 'set', 'par', 'var', or 'equ'.
+
+- One sheet per item.
+- Sets:
+
+  - Sheets for index sets have one column, with a header cell that is the set name.
+  - Sheets for one-dimensional indexed sets have one column, with a header cell that is the index set name.
+  - Sheets for multi-dimensional indexed sets have multiple columns.
+  - Sets with no elements are represented by empty sheets.
+
+- Parameters, variables, and equations:
+
+  - Sheets have zero (for scalar items) or more columns with headers that are the index *names* (not necessarily sets; see below) for those dimensions.
+  - Parameter sheets have 'value' and 'unit' columns.
+  - Variable and equation sheets have 'lvl' and 'mrg' columns.
```

Comment:
any particular reason to shorten names?

Reply:
That's what has always been returned by `Scenario.equ()` and `Scenario.par()`.